packages/cx-link/doc/Link.md

```diff
@@ -6,7 +6,8 @@ Link builds upon it, providing a sensible default combination of its generic tok
 and styles), to help meet typical site-use expectations.
 
 - Provides an Editable Link component;
-- Detects if the link is to a downloadable file, and adds an appropriate icon automatically;
+- Detects if the link points to an external URL or a downloadable file, and adds an appropriate icon
```

Comment:
@misterjones  in the content editor doc for this package, can you add if you link to an external link or downloadable file, for accessibility its best to add an aria label.  (Ticket is coming to try automate but don't know when it will be prioritized)